stringzilla/pybind11.cpp

```diff
@@ -115,7 +115,7 @@ index_span_t slice(size_t length, ssize_t start, ssize_t end) {
     ssize_t abs_start = std::abs(start);
     ssize_t abs_end = std::abs(end);
 
-    if (len == 0)
+    if (len == 0 || (start == 0 && end == 0))
```

Comment:
Isn't this better: `(len == 0 || start == end)`?

Reply:
Fixed.